src/doc/src/reference/unstable.md

```diff
@@ -1230,66 +1230,134 @@ cargo check -Z unstable-options -Z check-cfg-well-known-values
 
 ### workspace-inheritance
 
-* RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
-* Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
 
-The `workspace-inheritance` feature allows workspace members to inherit fields
-and dependencies from a workspace.
+### The `workspace.package` table
 
-Example 1:
+This would be in [`workspaces.md`][workspaces], under
+[The `workspace.metadata` table][the-workspacemetadata-table]
 
-```toml
-# in workspace's Cargo.toml
-[workspace.dependencies]
-log = "0.3.1"
-log2 = { version = "2.0.0", package = "log" }
-serde = { git = 'https://github.com/serde-rs/serde' }
-wasm-bindgen-cli = { path = "crates/cli" }
-```
+The `workspace.package` table is where you define keys that can be
+inherited by members of a workspace. These keys can be inherited by
+defining them in the member package with `{key}.workspace = true`.
+For Example:
 
 ```toml
```

Comment:
Nit: I would not have a newline between `For example:` and the example but before `For example:`